packages/dds/map/src/interfaces.ts

```diff
@@ -322,7 +322,6 @@ export interface ISerializedValue {
  * value is whatever params the ValueType needs to complete that operation.  Similar to ISerializableValue, it is
  * serializable via JSON.stringify/parse but differs in that it has no equivalency with an in-memory value - rather
  * it just describes an operation to be applied to an already-in-memory value.
- * @alpha
  */
 export interface IValueTypeOperationValue {
```

Comment:
@ChumpChief With the deprecation and removal of ValueTypes, can this interface be completely removed?

Reply:
Unfortunately no - Sequence still uses MapKernel to use valuetypes and we don't have a plan in place to change that.

Reply:
Yuck. OK. We'll make sense of this as part of API review. :)